templates/layout.html

```diff
@@ -15,6 +17,8 @@
             min-height: 100vh;
             /* height: 100%; */
         }
+        {% block style %}
+    {% endblock %}
```

Comment:
Don't add any additional CSS and make any changes in `templates/layout.html` directly. Doing this will break our all other pages.